airflow/api_connexion/endpoints/dataset_endpoint.py

```diff
@@ -120,3 +126,39 @@ def get_dataset_events(
     return dataset_event_collection_schema.dump(
         DatasetEventCollection(dataset_events=events, total_entries=total_entries)
     )
+
+
+@security.requires_access([(permissions.ACTION_CAN_CREATE, permissions.RESOURCE_DATASET)])
+@provide_session
+def post_dataset_event(session: Session = NEW_SESSION) -> APIResponse:
```

Comment:
Why can we only parse 1 event at the time?

Reply:
I followed the same pattern as dag_run. I think posting more than 1 dataset event at once may overcomplicate edge cases like a missing URI or a failure to register